doc/ref_model/chapters/chapter04.rst

```diff
@@ -296,19 +488,31 @@ The two :ref:`chapters/chapter02:profiles, profile extensions & flavours` are:
 ::
 
    Basic (B): for Workloads that can tolerate resource over-subscription and variable latency.
-   High Performance (H): for Workloads that require predictable computing performance, high network throughput and low network latency.
-
-The availability of these two (2) profiles will facilitate and accelerate workload deployment. The intent of the above profiles is to match the cloud infrastructure to the workloads most common needs, and allow for a more comprehensive configuration using profile-extensions when needed. These profiles are offered with `extensions <#profile-extensions>`__, that specify capability deviations, and allow for the specification of even more capabilities. The Cloud Infrastructure will have nodes configured as with options, such as virtual interface options, storage extensions, and acceleration extensions.
-
-The justification for defining these two profiles and a set of extensible profile-extensions was provided in Section :ref:`ref_model/chapters/chapter02:profiles, profile extensions & flavours` and includes:
-
--  Workloads can be deployed by requesting compute hosts configured as per a specific profile (Basic or High Performance)
--  Profile extensions allow a more granular compute host configuration for the workload (e.g., GPU, high, speed network, Edge deployment)
+   High Performance (H): for Workloads that require predictable computing performance, high network throughput and low
+   network latency.
+
+The availability of these two (2) profiles will facilitate and accelerate workload deployment. The intent of the above
+profiles is to match the cloud infrastructure to the workloads most common needs, and allow for a more comprehensive
+configuration using profile-extensions when needed. These profiles are offered with
+`extensions <#profile-extensions>`__, that specify capability deviations, and allow for the specification of even more
+capabilities. The Cloud Infrastructure will have nodes configured as with options, such as virtual interface options,
+storage extensions, and acceleration extensions.
+
+The justification for defining these two profiles and a set of extensible profile-extensions was provided in Section
+:ref:`ref_model/chapters/chapter02:profiles, profile extensions & flavours` and includes:
+
+-  Workloads can be deployed by requesting compute hosts configured as per a specific profile (Basic or High
+Performance)
+-  Profile extensions allow a more granular compute host configuration for the workload (e.g., GPU, high, speed network,
```

Comment:
no longer renders as a bullet and also subsequent bullets

Reply:
Fixed in [12d0ae2](https://github.com/cntt-n/CNTT/pull/2925/commits/12d0ae2dbdaec3632db8fc1d47864336a473d6be)